deploy/hub/managedcluster_escalation_clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: open-cluster-management:hub:escalation
+rules:
+  # Allow work agent to send event to hub
+  - apiGroups: ["", "events.k8s.io"]
```

Comment:
events is declared already in https://github.com/open-cluster-management/registration/blob/master/deploy/hub/hub_controller_clusterrole.yaml#L15, where i think we could add group of events.k8s.io

Reply:
added "events.k8s.io" group and tightened verbs by removing "list" and "watch" which is not in use by the recorder